docs/tutorials/mask_maps.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>To simplify maybe just work with one dataset? I guess you can also saveloy assume, that the PSF is largest at the lowest energy and just use 10 GeV...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3261/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
-ok to use only one dataset
-I agree that using the lowest energy here should be fine for the Fermi dataset
 but I prefer not use this assumption for clarity